clustering/web/hotrod/src/main/java/org/wildfly/clustering/web/hotrod/session/HotRodSessionMetaDataFactory.java

```diff
@@ -48,28 +48,23 @@
 import org.wildfly.clustering.web.session.ImmutableSessionMetaData;
 
 /**
+ * Factory for creating {@link SessionMetaData} backed by a pair of {@link RemoteCache} entries.
  * @author Paul Ferraro
+ * @param <C> the local context type
```

Comment:
Another s/C/L here - but we don't need to fix this now.

Reply:
Fixed